android-components/components/feature/session/src/main/java/mozilla/components/feature/session/SessionUseCases.kt

```diff
@@ -461,6 +463,58 @@ class SessionUseCases(
         }
     }
 
+    /**
+     * A use case for requesting a given tab's content be translated.
+     */
+    class TranslateUseCase internal constructor(
+        private val store: BrowserStore,
+    ) {
+        /**
+         * Request that Android translate the content of the current [tabId].
```

Comment:
```suggestion
         * Request to translate the content of the current [tabId].
```
I think we should drop `Android` wording here and below because we aren't technically having Android do the translation, but our translations engine(?).